File: EvenFibonacciNumbers/EFN.java
/*
Project Euler # 2

Brute Force
*/

public class EFN {
    public static void main(String[] args) {
        int a = 1, b = 2, temp = 0, sum = 0;

        while ( b < 4000000 ) {
            temp = b;
            b += a;
            a = temp;
            if ( a % 2 == 0 ) sum += a;
        }
        System.out.println(sum);
    }
}

File: EvenFibonacciNumbers/EFN.js
/*
Project Euler # 2

Brute Force
*/

var a = 1;
var b = 2;
var sum = b;

while (b < 4000000) {
    var temp = a;
    a = b;
    b = b + temp;
    if (b % 2 === 0) {
        sum += b;
    }
}

console.log(sum);
